homeassistant/components/lupusec/entity.py

```diff
@@ -0,0 +1,50 @@
+"""Provides the Lupusec entity for Home Assistant."""
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity import Entity
+
+from .const import DOMAIN, TYPE_TRANSLATION
+
+
+class LupusecDevice(Entity):
+    """Representation of a Lupusec device."""
+
+    _attr_has_entity_name = True
+    _attr_name = None
+
+    def __init__(self, data, device, entry_id) -> None:
+        """Initialize a sensor for Lupusec device."""
+        self._data = data
+        self._device = device
+        self._entry_id = entry_id
+        self._attr_unique_id = self.get_unique_id(
+            device.device_id if device.device_id != "0" else entry_id
+        )
```

Comment:
I think the separate function is a bit unneeded since every unique_id should be unique for the integration, so adding domain doesn't add much.

Reply:
Removed DOMAIN